.github/workflows/ci.yml

```diff
@@ -0,0 +1,257 @@
+            pkg_config_path: /usr/lib/s390x-linux-gnu/pkgconfig
+            variant: cross-compile
+
+          - container: "debian:stable"
+            arch: armhf
+            family: arm
+            compiler: arm-linux-gnueabihf-gcc
+            cross_compile: arm-linux-gnueabihf
+            pkg_config_path: /usr/lib/arm-linux-gnueabihf/pkgconfig
+            variant: cross-compile
+
+          - container: "debian:stable"
+            arch: arm64
+            family: aarch64
+            compiler: aarch64-linux-gnu-gcc
+            cross_compile: aarch64-linux-gnu
+            pkg_config_path: /usr/lib/aarch64-linux-gnu/pkgconfig
+            variant: cross-compile
+
+          - container: "debian:stable"
+            arch: ppc64el
+            family: ppc64
+            compiler: powerpc64le-linux-gnu-gcc
+            cross_compile: powerpc64le-linux-gnu
+            pkg_config_path: /usr/lib/powerpc64le-linux-gnu/pkgconfig
+            variant: cross-compile
+
+          - container: "debian:stable"
+            arch: s390x
+            family: s390x
+            compiler: s390x-linux-gnu-gcc
+            cross_compile: s390x-linux-gnu
+            pkg_config_path: /usr/lib/s390x-linux-gnu/pkgconfig
+            variant: cross-compile
+
+    container:
+      image: ${{ matrix.container }}
+      env:
+        ARCH: ${{ matrix.arch }}
+        FAMILY: ${{ matrix.family }}
+        CC: ${{ matrix.compiler }}
+        CROSS_COMPILE: ${{ matrix.cross_compile }}
+        PKG_CONFIG_PATH: ${{ matrix.pkg_config_path }}
+        VARIANT: ${{ matrix.variant }}
+
+    steps:
+    - name: Show OS
+      run: cat /etc/os-release
+
+    - name: Show env (matrix settings)
+      run: |
+        echo "ARCH: $ARCH"
+        echo "FAMILY: $FAMILY"
+        echo "CC: $CC"
+        echo "CROSS_COMPILE: $CROSS_COMPILE"
+        echo "VARIANT: $VARIANT"
+        echo "PKG_CONFIG_PATH: $PKG_CONFIG_PATH"
+
+    - name: Git checkout
+      uses: actions/checkout@v3
+
+    - name: Install additional packages
+      run: |
+        INSTALL=${{ matrix.container }}
+        INSTALL="${INSTALL%%:*}"
+        INSTALL="${INSTALL%%/*}"
+        ./ci/$INSTALL.sh
+        if [ "$VARIANT" ]; then ./ci/$INSTALL.$VARIANT.sh; fi
+
+    - name: Compiler version
+      run: $CC --version
+
+    - name: Display environment and Linux version
+      run: |
+        test -f /etc/issue && cat /etc/issue
+        echo "############################################"
+        lsb_release -a || true
+        echo "############################################"
+        cat /usr/include/linux/version.h
+        echo "############################################"
+        uname -a
+        echo "############################################"
+        cat /proc/cmdline
+        echo "############################################"
+        printenv
+
+    - name: Disable systemd
+      if: ${{ matrix.nosystemd == true }}
+      run: |
+        touch .nosystemd
```

Comment:
I'd say, drop this completely. The build system should be able to detect if systemd is present or not.

Reply:
*now should be able*